### ISSUE TITLE
google+ api shutting down on march 2019.

### ISSUE THREAD

**Issue by stratege1401** [nextcloud/socialsharing#28]
you might be concern !

> Hello Google+ Developer,
> 
> We’re writing to let you know that as part of the sunset of the consumer version of Google+, we will be shutting down our Google+ APIs on March 7, 2019. As part of these changes, Google+ Sign-in has been fully deprecated and will also be shut down on March 7, 2019.
> 
> What do I need to know?
> On March 7, 2019, all Google+ APIs and Google+ Sign-in will be shut down completely. This will be a progressive shutdown beginning in late January, with calls to these APIs starting to intermittently fail as early as January 28, 2019.

**Comment by e-alfred** [comment]
Given that Google+ will be shut down, the social sharing app should be deprecated anyway. But all the other platforms (especially Mail) should be supported on Nextcloud 15.

**Comment by stratege1401** [comment]
Coming back with more info ...

Google will shut down the OauthV1 key. At the time i reacted, I did not notice the V2 option for creating a new key.

I did follow the easy steps to fix my problem:
- create a new v2 key
- delete on my nextcloud the V1 key.
- add the v2 Key on my nextcloud.
- ( did this for all my v1 keys )
- delete on google dev console all my v1 key

For the last 11 days, i had not glitch with my NC15 apps. My google dev console stop complaining about v1 deprecation for mars 2019.

SO, i guess socialsharing app will be safe ( i dont use it, i use socialogin, same mechanics )...
